lib/chef/knife/bootstrap_windows_ssh.rb

```diff
@@ -106,7 +106,7 @@ def run_command(command = '')
         ssh.config[:identity_file] = config[:identity_file]
         ssh.config[:ssh_identity_file] = config[:ssh_identity_file] || config[:identity_file]
         ssh.config[:forward_agent] = config[:forward_agent]
-        ssh.config[:manual] = true
```

Comment:
needs to align with https://github.com/chef/chef/blob/master/lib/chef/knife/ssh.rb#L56